libindi/drivers/telescope/lx200driver.cpp

```diff
@@ -1283,7 +1284,8 @@ int Sync(int fd, char *matchedObject)
     if ((error_type = tty_write_string(fd, ":CM#", &nbytes_write)) != TTY_OK)
         return error_type;
 
-    error_type = tty_nread_section(fd, matchedObject, RB_MAX_LEN, '#', LX200_TIMEOUT, &nbytes_read);
+ //   error_type = tty_read_section(fd, matchedObject, '#', LX200_TIMEOUT, &nbytes_read);//zy2018
+ 	error_type = tty_read(fd,  matchedObject, 1, LX200_TIMEOUT, &nbytes_read);
```

Comment:
Explain the reason for this change as well.

Reply:
Command: “:CM#”
Response: “1

Reply:
Where did you get this from? The Meade documentation here says otherwise:

https://www.meade.com/support/LX200CommandSet.pdf

I can test it today as well with Meade at home.

Reply:
I refer to this: http://www.ioptron.com/v/ASCOM/RS-232_Command_Language2014_V2.0.pdf

Reply:
lx200driver.cpp is for Meade based scopes only. Where does iOptron make a call here?

Reply:
I use indi_lx200zeq25 to connect iOptron zeq25

Reply:
Ok then lx200zeq25 needs to implement its Sync function.. that's the solution.